infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/events/BrokerService.java

```diff
@@ -98,18 +98,29 @@ private void handle(BrokerStatusChangedEvent event) {
       LOG.error("Broker event skipped due to illegal content: {}", event);
       return;
     }
-    eventService.publish(new BrokerEvent(event, parseTooling(encodedTooling)));
-  }
-
-  @Nullable
-  private List<ChePlugin> parseTooling(String toolingString) {
-    if (!isNullOrEmpty(toolingString)) {
+    BrokerEvent brokerEvent;
+    List<ChePlugin> tooling = null;
+    if (!isNullOrEmpty(encodedTooling)) {
```

Comment:
Should we try to parse tooling even if the event contains an error?

Reply:
changed